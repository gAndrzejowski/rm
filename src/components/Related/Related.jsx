// @flow
import React from 'react';
import ResultUtils from '../ResultUtils/ResultUtils';
import Results from '../Results/Results';

const Related = () => (
  <article>
    <ResultUtils />
    <Results />
  </article>
);
export default Related;
