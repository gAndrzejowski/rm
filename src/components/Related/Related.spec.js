import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Related from './Related';

describe('Search Results component', () => {
    it('renders correctly', () => {
        const renderer = new ShallowRenderer();
        renderer.render(<Related />);
        expect(renderer.getRenderOutput()).toMatchSnapshot();
    })
});
