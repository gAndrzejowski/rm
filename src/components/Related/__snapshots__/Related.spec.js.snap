// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Results component renders correctly 1`] = `
<article>
  <Connect(ResultUtils) />
  <Connect(Results) />
</article>
`;
